README.md

```diff
@@ -6,16 +6,14 @@
 
 ## Introduction
 
-Python bindings for the Frequenz microgrid component graph rust library.
-
-TODO(cookiecutter): Improve the README file
+Python bindings for the [Frequenz microgrid component graph](https://github.com/frequenz-floss/frequenz-microgrid-component-graph-rs) rust library.
 
 ## Supported Platforms
 
 The following platforms are officially supported (tested):
 
-- **Python:** 3.11
-- **Operating System:** Ubuntu Linux 20.04
+- **Python:** 3.11, 3.12, 3.13, 3.14
+- **Operating System:** Ubuntu Linux 24.04, Microsoft Windows, Apple MacOS 15
```

Comment:
When looking at the CI mentioning `windows-latest`, I was wondering if it wasn't better to pin to a specific (set of) version(s), now seeing it in the README without any versions, it makes me thing again about that. I'm not sure how the situation is with compatibility between different Windows version, but maybe it would make more sense to pin and only specify the versions we are building for here?

Reply:
They only have windows server editions.  I think with windows you build on any, it runs on all supported.  And we don't need to mention that our stuff was built and tested on windows server only, probably makes no difference.

https://github.com/actions/runner-images

Also it is what maturin uses: https://github.com/PyO3/maturin/blob/f274d9a737d7ea2567cd4abe37f56d9ad9565570/src/ci.rs#L265-L268